charts/nginx-gateway-fabric/README.md

```diff
@@ -246,16 +246,18 @@ kubectl delete -f https://github.com/kubernetes-sigs/gateway-api/releases/downlo
 | `nginxGateway.image.repository`                         | The repository for the NGINX Gateway Fabric image.                                                                                                                                                       | ghcr.io/nginxinc/nginx-gateway-fabric                                                                           |
 | `nginxGateway.image.tag`                                | The tag for the NGINX Gateway Fabric image.                                                                                                                                                              | edge                                                                                                            |
 | `nginxGateway.image.pullPolicy`                         | The `imagePullPolicy` for the NGINX Gateway Fabric image.                                                                                                                                                | Always                                                                                                          |
 | `nginxGateway.lifecycle`                                | The `lifecycle` of the nginx-gateway container.                                                                                                                                                          | {}                                                                                                              |
 | `nginxGateway.extraVolumeMounts`                        | Extra `volumeMounts` for the nginxGateway container.                                                                                                                                                     | {}                                                                                                              |
 | `nginxGateway.gatewayClassName`                         | The name of the GatewayClass for the NGINX Gateway Fabric deployment.                                                                                                                                    | nginx                                                                                                           |
+| `nginxGateway.gatewayClassAnnotat`                      | Set of custom annotations for GatewayClass objects.                                                                                                                                                      | {}                                                                                                              |
```

Comment:
```suggestion
| `nginxGateway.gatewayClassAnnotations`                      | Set of custom annotations for GatewayClass objects.                                                                                                                                                      | {}                                                                                                              |
```

Reply:
Thanks for your help @salonichf5
good eye, fixed :)